plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixConfig.java

```diff
@@ -94,4 +101,17 @@ public PhoenixConfig setReuseConnection(boolean reuseConnection)
         this.reuseConnection = reuseConnection;
         return this;
     }
+
+    public Optional<Duration> getServerScanPageTimeout()
+    {
+        return serverScanPageTimeout;
+    }
+
+    @Config("phoenix.server-scan-page-timeout")
+    @ConfigDescription("Phoenix Page size to reflect the time limit on the amount of work single RPC request can do")
```

Comment:
Still feel bit confuse about the "Page size" in the description

Reply:
Sure let me fix it